### PR TITLE
Add arguments

### PR DESCRIPTION
**Pull request by archetipico**
Added some arguments and a manual (#56)

### CODE REVIEW

File: src/utils/trash.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ if [[ -d ~/.local/share/Trash/files ]]; then
 	printf "Should I clean Trash? "
 	read -p "[Y/n]: " ANSW
 	if [[ "$ANSW" == "y" ]]; then
-	    sudo rm -rf ~/.local/share/Trash/*
```
Comment (MasterCruelty) [inline src/utils/trash.sh:11]: Why you deleted that sudo? It's necessary to do a ```rm -rf ``` @archetipico

**Review by MasterCruelty — COMMENTED** [review]
I've seen a few changes you did. I'll check the rest tomorrow.

File: src/utils/trash.sh (hunk 1 above)
Comment (archetipico) [reply src/utils/trash.sh:11]: Some systems may not want `sudo`, and if you already inserted your password `rm -rf` should work without `sudo` anyway.

Comment (MasterCruelty) [reply src/utils/trash.sh:11]: > Some systems may not want `sudo`, and if you already inserted your password `rm -rf` should work without `sudo` anyway.

That's right, but you could lose privileges if updates take too much time

File: src/emerger.sh [hunk 2]
```diff
@@ -3,6 +3,12 @@
 SRC=$(dirname "$(readlink -f "$0")")
 source $SRC/utils/global.sh
 
+ARGV=$@
+if [[ $@ =~ "-help" ]]; then
+    cat $SRC/utils/help
```
Comment (MasterCruelty) [inline src/emerger.sh:8]: What ```$@ =~``` stands for?

**Review by MasterCruelty — COMMENTED** [review]
It looks ok. I dropped a comment on ```emerger.sh``` @archetipico

File: src/emerger.sh (hunk 2 above)
Comment (archetipico) [reply src/emerger.sh:8]: `If argument values contain`

Comment (MasterCruelty) [reply src/emerger.sh:8]: > `If argument values contain`

Ok so ```$@``` stands for the array of arguments you pass after ```up```

**Review by MasterCruelty — APPROVED** [review]
Ok I think it's all ok. I dropped a last comment on ```emerger.sh``` @archetipico

File: src/emerger.sh (hunk 2 above)
Comment (archetipico) [reply src/emerger.sh:8]: Indeed

Comment (MasterCruelty) [reply src/emerger.sh:8]: > Indeed

Okok, I'm gonna merge this changes

**Pull request merged**